### DISCUSSION TITLE
"One empowering phrase that sustains life."

### DISCUSSION THREAD

**Discussion by kazuki-hayashi-421** [NAIST-SE/AttractingContributors#823] [2023 Attraction Contributions]
Please share with me one phrase that you cherish in your life! 
The other day, my close friend got married. Even though I'm still a student🤣 
At this age, I find myself contemplating about the future. In such times, please give me a positive phrase to help me stay optimistic.

"Let's adorn our lives with eloquent expressions, enriching them with kindness and uplifting spirits! May your own journey be graced with boundless joy and contentment."

Come to my fork:
https://github.com/Queen-Vermouth/AttractingContributors

Add to my README file.

I’m looking forward to waiting for your words😊

**Comment by KKKKKXY** [comment]
I added my phrase into your list.(githubid: KKKKKXY)

**Reply by kazuki-hayashi-421** [reply]
Thank you for the nice word!!
I will live my life so as not to become an artist of this word!
I'll do yours later

**Comment by ton-na** [comment]
When I click link, 404 not found is displayed

**Reply by kazuki-hayashi-421** [reply]
Thanks for letting me know!!
I fixed it, please try it again!!
